Source/WebKit/GPUProcess/graphics/RemoteGraphicsContextGL.cpp

```diff
@@ -44,6 +45,8 @@
 #include "RemoteVideoFrameObjectHeap.h"
 #endif
 
+#define MESSAGE_CHECK(assertion) MESSAGE_CHECK_BASE(assertion, (m_gpuConnectionToWebProcess.get()->protectedConnection().ptr()))
```

Comment:
Ah, we didn't have a instance of message check here.
This is no good, as m_gpuConnectionToWebProcess is not thread-safe.